package.json

```diff
@@ -72,7 +72,8 @@
     "jsdoc": "3.6.4",
     "mustache": "4.0.1",
     "parse-strings-in-object": "2.0.0",
-    "rc": "1.2.8"
+    "rc": "1.2.8",
+    "uuid4": "2.0.2"
```

Comment:
What was the need for this?

Reply:
central-shared-services has uuid4 as a devDependency but it's used in the source code here https://github.com/mojaloop/central-services-shared/blob/bf68d0fd63e867cbe58fe610a9253e1cc82145bc/src/util/streaming/protocol/index.js

I plan to PR to move it to `dependencies`

Reply:
https://github.com/mojaloop/central-services-shared/pull/241

I made the PR but I added it here since I didn't want to wait for the next version of central-services-shared